docs/eventing/channels/create-default-channel.md

```diff
@@ -6,42 +6,61 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-# Creating a channel using cluster or namespace defaults
+Developers can create channels of any supported implementation type by creating an instance of a
+`Channel` object.
```

Comment:
If this is for mkdocs does it not need a H1?

Reply:
Also, be consistent with Channel vs `Channel` - in all the content above there are no backticks

Reply:
Yep, added the header, thanks. I'll drop the backticks from "Channel" as an object.